common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -572,6 +648,53 @@ innovation_chaosborn = {
 	flag = global_maa
 }
 
+innovation_draenei_tactics = {
```

Comment:
Unlocalized.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/151371991-89a4b5ba-2344-4d0e-b5c3-dd9437c2ce90.png)

</details>